app/src/main/java/com/example/daggersampleapplication/App.kt

```diff
@@ -1,10 +1,10 @@
 package com.example.daggersampleapplication
 
 import dagger.android.AndroidInjector
-import dagger.android.DaggerApplication
+import dagger.android.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mattukouta/DaggerSample/blob/3469102ef3fd38db1a6087a8e8bbbe4c3cbd7710/app/src/main/java/com/example/daggersampleapplication/App.kt#L4"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>